src/rdata/naptr.rs

```diff
@@ -0,0 +1,545 @@
+//! Record data from [RFC 3403]: NAPTR records.
+//!
+//! This RFC defines the Naptr record type.
+//!
+//! [RFC 3403]: https://tools.ietf.org/html/rfc3403
```

Comment:
While not consistently done throughout the domain code base yet, in newer code we have been opting to use rfc-editor.org links instead of ietf.org links.

```suggestion
//! [RFC 3403]: https://www.rfc-editor.org/info/rfc3403
```